src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1529,6 +1529,14 @@ def receive_timeseries_from_csv(
     if input_type == "input":
         compute_timeseries_properties(dict_asset)
 
+    if TIMESERIES in dict_asset:
+        if (dict_asset[TIMESERIES] == None).any():
+            incidents = sum(dict_asset[TIMESERIES] == None)
+            logging.Error(
+                f"A number of {incidents} NaN value(s) found in the {TIMESERIES} of {dict_asset[LABEL]}. Changing NaN value(s) to 0."
+            )
+            dict_asset.update({TIMESERIES: dict_asset[TIMESERIES].filna(0)})
+
```

Comment:
Hi @paragpatil39! Now that we have this code section, you should also write a pytest for it. I am not sure if there are already tests for the overarching function `C0.receive_timeseries_from_csv`. If there are, you can add a new test to the bunch where you provide a timeseries which includes NaNs. 

If there isnt, I would propose that you move this new code in a function and call that one like so:
```
receive_timeseries_from_csv():
      ...
      replace_nans_in_timeseries_with_0(dict_asset)
      
def replace_nans_in_timeseries_with_0(dict_asset):
    """
    Insert docstrings, see other functions for that
    """
    if TIMESERIES in dict_asset:
        if (dict_asset[TIMESERIES] == None).any():
            incidents = sum(dict_asset[TIMESERIES] == None)
            logging.Error(
                f"A number of {incidents} NaN value(s) found in the {TIMESERIES} of {dict_asset[LABEL]}. Changing NaN value(s) to 0."
            )
            dict_asset.update({TIMESERIES: dict_asset[TIMESERIES].filna(0)})
```
And then you can write a test for this new function.

Reply:
I first thought that a whole benchmark test would be necessary for this (thats why I let you create the `timeseries_nan` input folder), but I think this minimal example would also work.

If you want to have the feature tested in a benchmark test we can discuss it as well.

Please add a sentence at the bottom of [this section](https://multi-vector-simulator.readthedocs.io/en/latest/Model_Assumptions.html#input-verification) saying that NaNs in timeseries are replaced by 0.